common/jmeter/authenticate/Authenticate_Super_Tenant_User.jmx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jmeterTestPlan version="1.2" properties="4.0" jmeter="4.0 r1823414">
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
```

Comment:
shall we revert this jmeter version changes ?

AFIR,there was an issue with latest jmeter versions and we were using 4.0 in our tests

Reply:
Fixed with https://github.com/wso2/performance-is/pull/128/commits/9834b8cd5f7b6908c8e7b74316e1a94be8ab19ab